CMakeLists.txt

```diff
@@ -175,7 +179,7 @@ IF(PROJECT_DEBUG_POSTFIX)
   PKG_CONFIG_APPEND_CFLAGS_DEBUG(${FLAG_POSTFIX})
 ENDIF()
 
-SET(USE_CXX11 FALSE CACHE BOOL "Whether to use C++11")
+SET(USE_CXX11 TRUE CACHE BOOL "Whether to use C++11")
```

Comment:
This will hardcode the `-std=c++11` flag, breaking things for compilers which default to C++ >= 14

Reply:
Fixed with commit 6ee4ee5